polynote-frontend/package.json

```diff
@@ -22,6 +22,7 @@
   },
   "scripts": {
     "build": "webpack --config webpack.config.js",
+    "dist": "webpack --config webpack.config.js --mode production",
```

Comment:
might need to rebase on master after #173 is merged

Reply:
I'll just close #173 since it's included in this

Reply:
Oh, it automatically marked #173 as merged when I merged this.

:heart: github